DependencyInjection/DoctrineExtension.php

```diff
@@ -279,8 +279,16 @@ protected function getConnectionOptions($connection)
                     continue;
                 }
                 $options['master'][$key] = $value;
+                if (!isset($options['master']['driver'])) {
+                    $options['master']['driver'] = $options['driver'];
```

Comment:
doing this in each iteration of the loop does not make sense.

And why would you copy the driver ? AFAIK, it should not be copied to each connection